Open/store/open.ts

```diff
@@ -161,6 +167,7 @@ export const actions: ActionTree<OpenState, OpenState> = {
     async setInitialData ({ dispatch }, year) {
         await Promise.all([
             dispatch("setTournament", year),
+            dispatch("setTitle", year),
```

Comment:
```suggestion
            commit("setTitle", year),
```

Reply:
the setinitialdata function will need to add a dispatch to the args as well, see setTeam in actions